app/src/main/java/org/jellyfin/mobile/utils/Constants.kt

```diff
@@ -9,6 +9,7 @@ object Constants {
     const val APP_INFO_VERSION: String = BuildConfig.VERSION_NAME
 
     // Webapp constants
+    const val MINIMUM_WEB_VIEW_VERSION = 88
```

Comment:
Any particular reason version 88 was chosen?

Reply:
Tachiyomi uses the same, so it seemed sane.